### ISSUE TITLE
Tab Order Incorrect on Detail Views

### ISSUE THREAD

**Issue by jxthibeault** [jxthibeault/cot-install#49]
**Describe the bug**
Tab order skips the "Location" field on detail views

**To Reproduce**
1. Open a detail view for an installation order
2. Under primary equipment, tab through an equipment line

**Expected behavior**
Location should be inline in the tab order
